design/craft/inclusivecache/src/Parameters.scala

```diff
@@ -267,12 +268,12 @@ object MetaData
     ((opcode === TLMessages.AcquireBlock || opcode === TLMessages.AcquirePerm) && param =/= TLPermissions.NtoB)
   }
   // Does a request prove the client need not be probed?
-  def skipProbeN(opcode: UInt): Bool = {
+  def skipProbeN(opcode: UInt, hintsSkipProbe: Boolean): Bool = {
     // Acquire(toB) and Get => is N, so no probe
     // Acquire(*toT) => is N or B, but need T, so no probe
     // Hint => could be anything, so probe IS needed
```

Comment:
This comment need to be edited:
```suggestion
    // Hint => could be anything, so probe IS needed, if hintsSkipProbe is enabled, skip probe the same client
```